dev/DataTarget/NameNormalizer.php

```diff
@@ -16,6 +16,9 @@ public static function normalize(string $key): string
         }
 
         $key = str_replace([' ', ';', ',', '(', ')', '-', '.', '\'', '"', '/', '|', '=', '!', '?', '*', '[', ']', '~'], '_', $key);
+        if (preg_match('/[^0-9A-Z_a-z]/', $key) === 1) {
```

Comment:
This will break updating of existing specs that happens daily. I've added a workflow to also check the spec updating in PRs so the existing cases should be fixed before this can be merged. Can you update your branch with main?

Reply:
Done 🍏